sbin/decommission-slave.sh

```diff
@@ -40,6 +40,7 @@ fi
 if [ "$SPARK_WORKER_INSTANCES" = "" ]; then
   "${SPARK_HOME}/sbin"/spark-daemon.sh decommission org.apache.spark.deploy.worker.Worker 1
 else
+  echo "WARNING: Support of multiple workers on the same host is deprecated in Spark 3.0 and is going to be removed in Spark 3.1."
```

Comment:
I don't think this warning message is necessary, but we can consider add comment in the script to say the env `SPARK_WORKER_INSTANCES` is deprecated.